.github/workflows/edFi.ods.api manual.yml

```diff
@@ -0,0 +1,48 @@
+name: EdFi.Ods.Api Manually triggered build
+
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+
+env:
+  INFORMATIONAL_VERSION: "5.4"
+  BUILD_INCREMENTER: "10"
```

Comment:
This does not need to be 10 and instead could be 3, since the last build on TeamCity for this is 5.4.3, so we'd want the next in Github to be 5.4.4

Reply:
yes updated . Thanks